src/Subscriptions/BroadcastManager.php

```diff
@@ -54,6 +55,10 @@ protected function createPusherDriver(array $config): PusherBroadcaster
 
         $pusher = new Pusher($appKey, $appSecret, $appId, $options);
 
+        if ($driverConfig['log'] ?? false) {
```

Comment:
If this option is not documented, how are users supposed to know about it? Please add a default value and short description in `lighthouse.php`.

Reply:
this log key belongs to the `broadcasting` config file, not the lighthouse config. And if you see Laravel's config, that's also not even in the config and documented. https://github.com/laravel/laravel/blob/ebf2646c347b941e63709f7e69ab79416f6d5124/config/broadcasting.php#L33-L42

I found out while going through their code. So, thought to implement the same.

Reply:
I see, thank you.